File: 1/1.py
with open('input.txt', 'r') as file:
    text = file.read()
    numbers = list(map(int, filter(lambda x: len(x) > 0, text.split('\n'))))
    for i in numbers:
        for j in numbers:
            if i + j == 2020:
                print(i * j)

File: 13/1.jl
estimate, buses = eachline("input.txt")

estimate = parse(Int, estimate)
buses = parse.(Int, filter(!=("x"), split(buses, ",")))

earliest = buses[begin]

arrives(bus) = ceil(estimate / bus) * bus

for bus in buses
    if arrives(bus) < arrives(earliest)
        global earliest = bus
    end
end

println(round(Int, (arrives(earliest) - estimate) * earliest))

